yarn.lock

```diff
@@ -1692,6 +1692,13 @@
   dependencies:
     requireindex "~1.1.0"
 
+"@jest/schemas@^29.4.3":
+  version "29.4.3"
+  resolved "https://registry.yarnpkg.com/@jest/schemas/-/schemas-29.4.3.tgz#39cf1b8469afc40b6f5a2baaa146e332c4151788"
+  integrity sha512-VLYKXQmtmuEz6IxJsrZwzG9NvtkQsWNnWMsKxqWNu3+CnfzJQhp0WDDKWLVV9hLKr0l3SLLFRqcYHjhtyuDVxg==
+  dependencies:
+    "@sinclair/typebox" "^0.25.16"
+
```

Comment:
The commit added a devDpendency upon `tsd`. Almost any change in dependencies will change `yarn.lock`. Because that's automated there's not much to review. The important thing to review is the package.json changes. For yarn.lock you might want to verify there are no duplicates or that it hasn't grown dramatically. If there are duplicates they can be fixed with `npx yarn-deduplicate`.